ResearchKit/Common/ORKQuestionStepViewController.m

```diff
@@ -202,6 +204,7 @@ - (void)stepDidChange {
             _questionView.headerView.learnMoreButtonItem = self.learnMoreButtonItem;
             _questionView.continueSkipContainer.skipButtonItem = self.skipButtonItem;
             _questionView.continueSkipContainer.continueEnabled = [self continueButtonEnabled];
+            _questionView.continueSkipContainer.hidden = self.parentReviewStep;
```

Comment:
This line is evaluated as `... = (self.parentReviewStep != nil)`. I can make this clearer if you'd like